### PR TITLE
Add CF.RESERVE example

### PR DESCRIPTION
**Pull request by dnielsen**
add CF.RESERVE example and changed key name from newFilter to newCuckooFilter

### CODE REVIEW

**Comment by K-Jo** [comment]
@dnielsen I would leave this as is and stick to the error messages all redis users are used to.

**Comment by K-Jo** [comment]
@dnielsen can you please remove the line with the change of error message so we can progress?

File: docs/Quick_Start.md [hunk 1]
```diff
@@ -83,18 +91,18 @@ You can add the item multiple times. The filter will attempt to count it.
...
-127.0.0.1:6379> CF.EXISTS newFilter notpresent
+127.0.0.1:6379> CF.EXISTS newCuckooFilter notpresent
 (integer) 0
 ```
 
 ## Cuckoo: Deleting item from filter
 
 ```
-127.0.0.1:6379> CF.DEL newFilter foo
+127.0.0.1:6379> CF.DEL newCuckooFilter foo
```
Comment (ashtul) [inline docs/Quick_Start.md]: We don't currently have CF.DEL. Did you mean to write DEL?

File: src/redismodule.h [hunk 2]
```diff
@@ -63,7 +63,7 @@
 #define REDISMODULE_HASH_DELETE ((RedisModuleString*)(long)1)
 
 /* Error messages. */
-#define REDISMODULE_ERRORMSG_WRONGTYPE "WRONGTYPE Operation against a key holding the wrong kind of value"
+#define REDISMODULE_ERRORMSG_WRONGTYPE "WRONGTYPE Operation against a key holding the wrong kind of value or key already exists"
```
Comment (ashtul) [inline src/redismodule.h]: We should not change this but rather separate the cases and return the appropriate message for each one of them. Please remove.

**Pull request merged**